policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/additional.yaml

```diff
@@ -11,6 +11,8 @@ metadata:
      # Legal code specifies $1,000 with inflation adjustments.
     - title: Legal Code Titl. 32 V.S.A. § 5811(21)(C)(iii)
       href: https://legislature.vermont.gov/statutes/section/32/151/05811
-
+    - title: 2023 VERMONT Income Tax Return Form Instructions-Taxable Income Line 4
+      href: https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf#page=10 
 values:
   2021-01-01: 1_050
```

Comment:
could you check 2022 value if it changed?

If so please add:
add to the metadata:
uprating: gov.irs.uprating